src/components/AssetManagementPanel/AssetManagementApplication/index.tsx

```diff
@@ -27,10 +28,34 @@ export const AssetManagementApplication = ({
   );
   const { send: sendSurrender, state: surrenderingState } = useContractFunctionHook(titleEscrow, "transferTo");
 
+  const { send: changeHolder, state: holderState } = useContractFunctionHook(titleEscrow, "changeHolder");
+
   const onSurrender = () => {
     sendSurrender(tokenRegistryAddress);
   };
 
+  const onTransfer = (nextHolder: string) => {
+    changeHolder(nextHolder);
+  };
+
+  const onSetFormAction = (AssetManagementActions: AssetManagementActions) => {
+    setAssetManagementAction(AssetManagementActions);
+    setHolderErrorState("");
+  };
+
+  useEffect(() => {
+    setHolderErrorState(holderState);
```

Comment:
Why are you setting the state into error state?

Reply:
so that when i click back or cancel, the error state will disappear. Than when i click transfer holder again it will not show the error.